docs/current_docs/api/module-dependencies.mdx

```diff
@@ -107,3 +109,39 @@ dagger install ./path/to/module
 :::note
 Installing a module using a local path (relative or absolute) is only possible if your module is within the repository root (for Git repositories) or the directory containing the `dagger.json` file (for all other cases).
 :::
+
+## Uninstallation
+
+To remove a dependency from your Dagger module, use the `dagger uninstall` command, as in the following example:
+
+```sh
+dagger uninstall github.com/shykes/daggerverse/hello
```

Comment:
I think *probably* we should suggest `dagger uninstall hello` and suggest uninstalling by name first?

Even though we have compatibility with `source` everywhere, I think looking at dependencies by name is shorter and cleaner.

Reply:
Agree!

Reply:
Updated